packages/one-app-locale-bundler/CHANGELOG.md

```diff
@@ -1,134 +0,0 @@
-# Change Log
```

Comment:
What about the Release notes that were duplicated and deleted, will semantic release generate new ones?

Reply:
I've removed them and created a main changelog. Semantic release will create a new changelog e.g https://github.com/americanexpress/one-app-cli/blob/beta/packages/eslint-plugin-one-app/CHANGELOG.md